artifactory/cli.go

```diff
@@ -1486,10 +1485,16 @@ func shouldSkipGoFlagParsing() bool {
 	}
 
 	if !exists {
-		return false
+		// If missing in the root project, check in the home dir
+		jfrogRootDir, err = config.GetJfrogHomeDir()
+		if err != nil {
+			cliutils.ExitOnErr(err)
+		}
+	} else {
+		jfrogRootDir = filepath.Join(jfrogRootDir, ".jfrog")
 	}
 
-	configFilePath := filepath.Join(jfrogRootDir, ".jfrog", "projects", "go.yaml")
+	configFilePath := filepath.Join(jfrogRootDir, "projects", "go.yaml")
```

Comment:
It looks like there a few places in the code which construct the **<project dir>/project/go.yaml* similarly. Maybe we should consolidate all of them into one function.
In addition, notice that the logic of finding the go.yaml file (searching for it upstream and then in the home dir) is duplicated in the *goCmd* and *shouldSkipGoFlagParsing* functions.

Reply:
Done I've created one function that both goCmd and shouldSkipGoFlagParsing are using